### ISSUE TITLE
Produced plots in for loop get suppressed 

### ISSUE THREAD

**Issue by Abdelgha-4** [sfu-db/dataprep#679]
**Describe the bug**
When trying to produce many plots using the `plot` function inside a loop, none of them is shown.

**To Reproduce**
I found that this bug can be seen even in the official documentation!
![image](https://user-images.githubusercontent.com/72174073/130576237-40f3dfea-4137-4b2d-90b4-4a812f234740.png)

**Desktop :**
 - OS: Windows10
 - Browser chrome
 - Python Version     : 3.8.11
 - Dataprep Version     : 0.2.15
 - Additional:
     - jupyter core     : 4.7.1
     - jupyter-notebook : 6.4.1
     - qtconsole        : 5.1.0
     - ipython          : 7.26.0
     - ipykernel        : 5.3.4
     - jupyter client   : 6.1.12

**Comment by Abdelgha-4** [comment]
I just realized that to fix that it just needed to add `.show()` after each `plot()` call, be cause without it the `plot` function just return the plot object, however I'll let this issue opened since the problem is still in the documentation, until it get fixed, and I think this shouldn't be tagged as a 'bug' anymore.

**Comment by Abdelgha-4** [comment]
just to be more clear, the following code should produce the desired results in the first example :

```
for feature in ['Age', 'Cabin', 'Embarked']:
    plot(train_df, feature, 'Survived').show()
```

[link to the documentation page with the issue](https://docs.dataprep.ai/user_guide/eda/titanic.html)

**Comment by jinglinpeng** [comment]
Hi @Abdelgha-4 , thanks a lot for the detailed report and link! I've fixed the doc in #680 . It will be built after another bug fixed.
